### DISCUSSION TITLE
temp files of canceled plot cannot be deleted

### DISCUSSION THREAD

**Discussion by irmakakyeli** [Chia-Network/chia-blockchain#3040] [Q&A]
Hello, I canceled a plot and I want to empty my temp file however it does not let me delete them saying Chia is using this file even if the program is closed.

**Comment by dorkmo** [comment]
the plotter is likely still running in that background even though gui has stopped. you'll need to kill the process or simply restart your computer.

**Reply by irmakakyeli** [reply]
thank you!
